src/pages/MyPage/EditProfession.js

```diff
@@ -1,31 +1,52 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
+import axios from 'axios';
 import { CATEGORY } from './PROFESSION_CATEGORY';
+import { SUBCATEGORY_ID, CAREER_ID, CAREER_ID_TO_STR } from './MYPAGE_IDS';
 
-const EditProfession = ({ setIsProfessionEdit }) => {
-  const [selectedOptions, setSelectedOptions] = useState({
-    category: '',
-    subcategory: '',
-    years: '',
-    salary: '',
-  });
+const EditProfession = ({ info, setIsProfessionEdit, token, setIsUpdated }) => {
+  const initialSelectedOptions = {
+    category: `${info.job_category ? info.job_category : ''}`,
+    subcategory: `${info.job_subcategory ? info.job_subcategory : ''}`,
+    years: `${info.career ? CAREER_ID_TO_STR[info.career] : ''}`,
+    salary: `${info.salary ? info.salary : ''}`,
+  };
+
+  const [selectedOptions, setSelectedOptions] = useState(
+    initialSelectedOptions
+  );
 
   const handleSelectBoxChange = e => {
     setSelectedOptions({ ...selectedOptions, [e.target.name]: e.target.value });
   };
 
-  // TODO : POST 명령 들어와야함
-  const clickSaveBtn = () => {
-    fetch('API 주소 넣어야함', {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-        Authorization:
-          sessionStorage.getItem('세션스토리지에 저장될 토큰 이름!'),
-      },
-      // FIXME 여러 계정에 대한 정보가 담긴 객체에서 profession 부분만 업데이트 되어야하는데...
-      body: JSON.stringify(selectedOptions),
-    });
+  const updateProfessionInfo = () => {
+    if (selectedOptions.subcategory === '') {
+      return alert('직무를 선택해주세요!');
+    }
+
+    const headers = {
+      Authorization: token,
+    };
+
+    const body = {
+      ...info,
+      occupation: SUBCATEGORY_ID[selectedOptions.subcategory],
+      career: CAREER_ID[selectedOptions.years],
+      salary: selectedOptions.salary,
+    };
+
+    axios
+      .post('http://15.165.203.121:8080/users/profile', body, {
+        headers: headers,
```

Comment:
```suggestion
        headers
```

단축속성명 사용해도 되는 케이스 같네요!